### PR TITLE
Save video-16.svg, video-24.svg

### PR DESCRIPTION
**Pull request by edokoa**
Adding `video` icon

### CODE REVIEW

File: icons/video-16.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><path fill-rule="evenodd" d="M1.75 3.5a.25.25 0 00-.25.25v8.5c0 .138.112.25.25.25h12.5a.25.25 0 00.25-.25v-8.5a.25.25 0 00-.25-.25H1.75zM0 3.75C0 2.784.784 2 1.75 2h12.5c.966 0 1.75.784 1.75 1.75v8.5A1.75 1.75 0 0114.25 14H1.75A1.75 1.75 0 010 12.25v-8.5z"/><path d="M6 10.559V5.442a.25.25 0 01.379-.215l4.264 2.559a.25.25 0 010 .428l-4.264 2.559A.25.25 0 016 10.559z"/></svg>
```
Comment (Juliusschaeper) [inline icons/video-16.svg:1]: Looks good to me.

File: icons/video-24.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M1.75 4.5a.25.25 0 00-.25.25v14.5c0 .138.112.25.25.25h20.5a.25.25 0 00.25-.25V4.75a.25.25 0 00-.25-.25H1.75zM0 4.75C0 3.784.784 3 1.75 3h20.5c.966 0 1.75.784 1.75 1.75v14.5A1.75 1.75 0 0122.25 21H1.75A1.75 1.75 0 010 19.25V4.75z"/><path d="M9 15.584V8.416a.5.5 0 01.77-.42l5.576 3.583a.5.5 0 010 .842L9.77 16.005a.5.5 0 01-.77-.42z"/></svg>
```
Comment (Juliusschaeper) [inline icons/video-24.svg:1]: Looks good to me.

**Review by Juliusschaeper — COMMENTED** [review]
Looks good

**Review by ashygee — APPROVED** [review]
🚀

**Comment by edokoa** [comment]
This is the PR for https://github.com/github/design-systems/issues/1155

//cc @ohjoycelau

**Comment by ohjoycelau** [comment]
🚀 🚀 🚀 

https://github.com/github/design-systems/issues/1155#issuecomment-769267080

> Tested in a video block and some other fake instances. Looking good to me 👍 thanks!
>
> Screen Shot 2021-01-28 at 9 56 12 AM

**Pull request merged**